com.arviceblot.eso-addon-manager.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.arviceblot.eso-addon-manager",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "eso-addon-manager",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
```

Comment:
`host` is a very broad filesystem permission

Reply:
This is for various source game install location options. They could be installed anywhere, but directory is selected by user. The original intent is to run on Steam Deck where games can be moved from SSD to SD Card, etc.